web_src/css/modules/animations.css

```diff
@@ -35,6 +35,19 @@
   border-width: 2px;
 }
 
+form.is-loading::after {
+  max-height: 50%;
+  max-width: 50%;
+}
+
+form.single-button-form.is-loading > * {
```

Comment:
Literally, it's not only for `single-button-form`, but it's for "hide content when loading" (eg: `loading-hide-content`) , including `.markup`

Reply:
It's not completely hiding content, just the content of a button, but not the border. I'm not sure if a general solutions exist. It would have been easier if there were no form, but just the button.

Reply:
It should be in both preview tab and regular view.

Reply:
@wxiaoguang are you okay when we keep this? Problem is for single form button, the loading state should be on the button, but not go semi-transparent, just replace the text on the button with the loader. I can't think of any clean way to do this with CSS, not even `:has` selector can count occurences of `button` contained in the form.

Reply:
Added the comment to code